### ISSUE TITLE
 Cannot read property 'detail' of undefined

### ISSUE THREAD

**Issue by vscodeerrors** [Microsoft/vscode#3231]
Issue Id: <b>70b9c65b-9ccd-16b5-73e0-508ddd36a5cc</b><br /><br />*Versions* <br />- 0.10.9<br />-  db71ac615ddf9f33b133ff2536f5d33a77d4774e<br />*Stack* 

```
/vs/platform/telemetry/common/abstractTelemetryService.ts#73:7 (AbstractTelemetryService.onErrorEvent) 
/vs/base/common/errors.ts#55:3 (ErrorHandler.emit) 
/vs/base/common/errors.ts#54:17 (ErrorHandler.emit) 
/vs/base/common/errors.ts#73:7 (ErrorHandler.onUnexpectedError) 
/vs/base/common/errors.ts#87:15 (onUnexpectedError) 
/vs/workbench/node/pluginHostProcess.ts#56:6 (connectToRenderer) 
```

**Comment by alexandrudima** [comment]
@isidorn Is something not working with stack deminification?

@joaomoreno Not sure if I had to do something more to upload the sourcemaps for 0.10.9?

**Comment by isidorn** [comment]
I forgot we have a new version, I will upload the sourcemaps

**Comment by joaomoreno** [comment]
Thanks @isidorn

**Comment by isidorn** [comment]
Actually Joao will upload it. @alexandrudima feel free to close this

**Comment by joaomoreno** [comment]
Don't close this.

**Comment by joaomoreno** [comment]
I just published the sourcemaps for db71ac6, but it doesn't seem to have worked.

Moving to @SofianHn

**Comment by joaomoreno** [comment]
Ok, got the stack. Updated initial comment.

Not sure if should assign it to @SofianHn or @alexandrudima .
